examples/kops-with-vpc/README.md

```diff
@@ -0,0 +1,184 @@
+# Example to test the Kops workflow with an existing VPC on AWS
+
+## Summary
```

Comment:
I think this summary could use an explanation of the motivation for having Terraform handle creating the VPC rather than letting `kops` do that itself.  Basically: this makes it easier to fit a cluster into your existing infrastructure (e.g. you might want the same VPC to container other non-kops resources).